build/NuGetPackages/Microsoft.Build.Tasks.Core.nuspec

```diff
@@ -67,6 +67,7 @@
         <dependency id="Microsoft.Build.Utilities.Core" version="[$version$]" />
         <dependency id="System.Collections.Immutable" version="1.3.1" />
         <dependency id="System.Reflection.Metadata" version="1.3.0" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" />
```

Comment:
Yes, and the nuget package 4.3.0 brings the 4.0.1.0 RuntimeInformation assembly. I had initially written 4.0.1.0 in all the jsons and then spent half an hour trying to reason why didn't get it :)